game-core/src/main/java/games/strategy/engine/framework/map/download/MapDownloadProgressListener.java

```diff
@@ -73,12 +74,13 @@ private void updateProgressBarWithPercentComplete(
 
   void downloadUpdated(final long currentLength) {
     final String toolTipText = String.format("Installing to: %s", download.getInstallLocation());
-    OptionalUtils.ifPresentOrElse(
```

Comment:
I just realised we no longer need this helper method at all, this is a method of Optional that was introduced in a java version 9-11

Reply:
Indeed, it's nice to see the optional API expanded as much as it has.